test/test_functional.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+import torch
+import torchaudio
+import unittest
+import test.common_utils
+
+
+class TestFunctional(unittest.TestCase):
+    # size (2,20)
+    test_data = torch.tensor([
+        [45.4243, 81.9316, 19.1100, 32.4998, 45.3313, 68.8204, 42.0782, 19.7222,
+         76.8721, 69.9104, 27.7188, 86.3579, 30.3251, 92.0308, 70.0568, 74.8940,
+         94.3127, 82.9875, 88.8303, 96.3460],
+        [59.4262, 91.0040, 74.7672, 79.8533, 46.7943, 13.6757, 85.5145, 33.0060,
+         88.5102, 25.6912, 57.9501, 33.3326, 71.5654, 90.0321, 81.8218, 91.6907,
+         87.9834, 16.4177, 62.4474, 0.2146]
+    ]).float()
+
+    def _test_istft_helper(self, sound, kwargs):
+        stft = torch.stft(sound, **kwargs)
```

Comment:
Don't only rely on this invariant. Also add some simpler sanity checks / edge cases. If stft and istft fail because a common component fails in a similar manner (say something like conv1d) then they could both produce garbage, but still be the inverse of the other.